onnxruntime/core/providers/openvino/ov_versions/data_ops.cc

```diff
@@ -262,6 +264,10 @@ void DataOps::populate_types_supported() {
       std::make_pair(V_2021_4, ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_INT8));
   supported_types_initializer_.insert(
       std::make_pair(V_2021_4, ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_UINT8));
+  supported_types_initializer_.insert(
+      std::make_pair(V_2024_4, ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_INT4));
```

Comment:
INT4 support started in 2024_5

Reply:
> INT4 support started in 2024_5

As per below PR the support for INT4 started from OV v2024.4 see the milestone section

https://github.com/openvinotoolkit/openvino/pull/24273


Thus I'm retaining my current changes here

Reply:
But you have it tested in 2024.5 , If you can show me testing proof with 2024.4 then you can put 2024.4 here

Reply:
Resolving as discussed earlier.